### PR TITLE
Refactor pzmm

### PR DESCRIPTION
**Pull request by smlindauer**

### CODE REVIEW

File: src/sasctl/pzmm/pickleModel.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-import os
+from pathlib import Path
 
 import pickle
 
 # %%
 class PickleModel():
     
-    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=os.getcwd()):
+    def pickleTrainedModel(trainedModel, modelPrefix, pPath=Path.cwd()):
```
Comment (jlwalke2) [inline src/sasctl/pzmm/pickleModel.py:13]: def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=Path.cwd()):

File: src/sasctl/pzmm/zipModel.py [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
 from pathlib import Path
 import zipfile
+import io
 
 # %%
 
 class ZipModel():
     
-    def zipFiles(self, fileDir, modelPrefix):
+    def zipFiles(fileDir, modelPrefix):
```
Comment (jlwalke2) [inline src/sasctl/pzmm/zipModel.py:13]: def zipFiles(self, fileDir, modelPrefix):

**Review by jlwalke2 — CHANGES_REQUESTED** [review]

File: src/sasctl/pzmm/zipModel.py (hunk 2 above)
Comment (smlindauer) [reply src/sasctl/pzmm/zipModel.py]: Fixed in [07ad6ce](https://github.com/sassoftware/python-sasctl/pull/76/commits/07ad6ce2532e24b1ef56e54c5b7ec572621f1fb0)

File: src/sasctl/pzmm/pickleModel.py (hunk 1 above)
Comment (smlindauer) [reply src/sasctl/pzmm/pickleModel.py]: Fixed in [07ad6ce](https://github.com/sassoftware/python-sasctl/pull/76/commits/07ad6ce2532e24b1ef56e54c5b7ec572621f1fb0)

**Review by jlwalke2 — APPROVED** [review]

**Pull request merged**